compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaMethodSymbol.java

```diff
@@ -60,7 +60,7 @@ public FunctionTypeSymbol typeDescriptor() {
 
     @Override
     public Optional<String> getName() {
-        return this.functionSymbol.getName();
+        return this.functionSymbol.getName().map(this::escapeReservedKeyword);
```

Comment:
This is not necessary right? IINM, the name would already be escaped in the result returned by `functionSymbol.getName()`.

Reply:
Fixed.